controls/V-73307.rb

```diff
@@ -56,13 +56,25 @@
   servers will synchronize with an authorized time server in the hierarchy."
   is_domain = command('wmic computersystem get domain | FINDSTR /V Domain').stdout.strip
 
-  describe command(' W32tm /query /configuration | Findstr Type') do
-    its('stdout') { should eq "Type: NT5DS (Local)\r\n" }
-  end if is_domain != 'WORKGROUP'
+  if is_domain != 'WORKGROUP'
+    pdc_emulator = command('netdom query fsmo | findstr PDC').stdout.split[1]
+    hostname = command('wmic computersystem get DNSHostName | findstr /V DNSHostName').stdout.strip
+    if pdc_emulator != hostname + "." + is_domain
+      describe command(' W32tm /query /configuration | Findstr Type') do
+        its('stdout') { should eq "Type: NT5DS (Local)\r\n" }
+      end
+    else
+      impact 0.0
+      desc 'This system is a domain controller with the PDC Emulator role, therefore this control is not applicable.'
```

Comment:
We don't need to update the main description of the control. The `skip_message` will come through.

Reply:
Removed `desc` line.